hack/ensure-kind.sh

```diff
@@ -42,8 +42,8 @@ verify_kind_version() {
   fi
 
   local kind_version
-  kind_version=$(kind version)
-  if ! [[ "${kind_version}" =~ ${MINIMUM_KIND_VERSION} ]]; then
+  kind_version="v$(kind version -q)"
+  if [[ "${MINIMUM_KIND_VERSION}" != $(echo -e "${MINIMUM_KIND_VERSION}\n${kind_version}" | sort -s -t. -k 1,1n -k 2,2n -k 3,3n | head -n1) ]]; then
```

Comment:
Won't `sort -V` be better than `sort -s -t. -k 1,1n -k 2,2n -k 3,3n`?

Reply:
Ah, other scripts use this fancy sort command, I guess it's fine then :smile:

Reply:
Right. I would also prefer a simpler version `sort -V`.
If we always use Linux, then `sort -V` is fine.

But, if we are to care about non-Linux systems, then we are not sure `sort -V` is always ok.
That's why I just followed the way how existing scripts are implemented.